namer/consul/src/test/scala/io/buoyant/namer/consul/ConsulNamerTest.scala

```diff
@@ -51,7 +51,7 @@ class ConsulNamerTest extends FunSuite with Awaits {
       Future.value(LocalAgent(Config = Some(Config(Domain = Some(domain), Datacenter = datacenter))))
   }
 
-  test("Namer stays pending while looking up datacenter for the first time") {
+  test("Namer returns Neg while looking up datacenter for the first time") {
```

Comment:
I don't think the updated test name is correct.  It looks like this tests that the state remains pending while waiting for a response from consul.